linux/net/rina/ipcps/shim-eth-vlan.c

```diff
@@ -1227,8 +1437,38 @@ static int eth_vlan_assign_to_dif(struct ipcp_instance_data * data,
                                 data->dif_name = NULL;
                                 return -1;
                         }
+                } else if (!strcmp(entry->name, "qdisc-max-size")) {
+                	ASSERT(entry->value);
+
+                	result = kstrtouint(entry->value, 10, &temp);
+                	if (result) {
+                		LOG_ERR("Cannot conver qdisc-max-size to uint");
```

Comment:
Be careful when trying to fix this. The string hall NOT be shortened. 

String traces cannot be split, to make them grepable. Check this exception in kernel's ciding style.